examples/knative-eventing-tutorial/README.md

```diff
@@ -0,0 +1,21 @@
+# Knative Eventing Tutorial
+See [TMP: Knative Eventing Tutorial](https://github.com/ease-lab/vhive/wiki/TMP:-Knative-Eventing-Tutorial) on the Wiki for a primer on Knative Eventing.
```

Comment:
please don't refer to TMP-marked wiki pages (in fact, avoid referring to any wiki page, wiki is a scratchpad here). refer to docs/ if necessary

Reply:
Should I move the wiki page to docs/ then or not refer to it at all?

Reply:
to `docs/knative/eventing.md`. please ask Mike to review it first though. but before that, please finalize the working example's code